manifest.json

```diff
@@ -3,8 +3,7 @@
     "browser_specific_settings": {
         "gecko": {
             "id": "changequote@caligraf",
-            "strict_min_version": "128.0",
-            "strict_max_version": "129.*"
+            "strict_min_version": "136.0"
```

Comment:
Thunderbird seems to need a `strict_max_version` as well, in addition to `strict_min_version`.  Without this, it throws an error: "This add-on could not be installed because it appears to be corrupt."